doc/releases/nautilus.rst

```diff
@@ -216,13 +216,24 @@ Instructions
         "ceph version 14.2.0 (...) nautilus (stable)": 22,
      }
 
-#. Scan for any OSDs deployed with the old ceph-disk tool to ensure
-   that ceph-volume can activate them after a host reboot.  On each
-   host containing OSDs,::
+#. If there are any OSDs in the cluster deployed with ceph-disk (e.g.,
+   almost any OSDs that were created before the Mimic release), you
+   need to tell ceph-volume to adopt responsibility so they can be
+   activated after a host reboot.  On each host containing OSDs,::
 
      # ceph-volume simple scan
      # ceph-volume simple activate --all
 
+   We recommend that each OSD host be rebooted following this step to
+   verify that the OSDs start up automatically.
+
+   Note that ceph-volume doesn't have the same hot-plug capability
+   that ceph-disk did, where a newly attached disk is automatically
+   detected via udev events.  If a ceph-disk-based OSD is moved to a
+   new host, or the host OSD is reinstalled, or the ``/etc/ceph/osd``
+   directory is lost, you will need to repeat the above commands to
```

Comment:
```suggestion
   directory is lost, you will need to scan the device *directly*  to
```

Reply:
scanning the device directly will presumably require a different command - should that command be mentioned here?

Reply:
It is the same command but with the device path, so instead of:

    ceph-volume simple scan

It would be:

    ceph-volume simple scan /dev/device

I guess it can't hurt?